packages/react-scripts/template-typescript/public/index.html

```diff
@@ -25,8 +23,12 @@
     <title>React App</title>
   </head>
   <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
+    <noscript>
+      You need to enable JavaScript to run this app.
+    </noscript>
+    <main>
```

Comment:
Is it possible to put `#root` on `main` instead of having an extra DOM node?

Reply:
it's not a good idea to use `main` here. According to W3C spec we should have only one main per document. CRA should have an opinion on what's the main content of the page.